DopplerCustomDomain/CustomDomainProvider/CustomDomainProviderService.cs

```diff
@@ -0,0 +1,35 @@
+using Flurl.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Tavis.UriTemplates;
+
+
+namespace DopplerCustomDomain.CustomDomainProvider
+{
+    public class CustomDomainProviderService : ICustomDomainProviderService
+    {
+        private readonly ILogger<CustomDomainProviderService> _logger;
+
+        public CustomDomainProviderService(ILogger<CustomDomainProviderService> logger)
+        {
+            _logger = logger;
+        }
+
+        public Task<List<string>> CreateAllCustomDomains()
+        {
+            throw new System.NotImplementedException();
+        }
```

Comment:
What is this?

Reply:
This is a method on the [CustomDomainService](https://github.com/MakingSense/doppler-forms/blob/0a52542914104ad55e68b8bb0b743d5b62521288/DopplerForms.Services/ICustomDomainService.cs) of Doppler Forms

Reply:
I understand now. It is because Doppler Forms has access to our database and it can query our DB to search for all custom domains. 

But I think that this new API should not depend on our databases, it should be isolated. So, in my opinion, this services do not fit here.

Reply:
If you want to create all the Relay custom domains, you should call this API one by one, manually, or from a Relay worker.